server/controllers/stripe.js

```diff
@@ -0,0 +1,96 @@
+const stripe = require('stripe')(process.env.STRIPE_SECRET_KEY);
+
+const url = 'https://localhost:3000/edit_profile';
```

Comment:
you can consider storing an env var called `CLIENT_DOMAIN=https://localhost:3000` so if you ever do deploy this, you can easily control between a domain vs localhost :)

Reply:
Updated on my current local working branch. Will be reflected in the next PR. Thanks!